cdap-common/src/test/java/co/cask/cdap/common/queue/QueueNameTest.java

```diff
@@ -27,32 +28,35 @@
   @Test
   public void testQueueNameForFlowlet() {
     // create a queue name
-    QueueName queueName = QueueName.fromFlowlet("app", "flow", "flowlet", "out");
+    QueueName queueName = QueueName.fromFlowlet(Constants.DEFAULT_NAMESPACE, "app", "flow", "flowlet", "out");
 
     // verify all parts are correct
     Assert.assertTrue(queueName.isQueue());
     Assert.assertFalse(queueName.isStream());
-    Assert.assertEquals("app", queueName.getFirstComponent());
-    Assert.assertEquals("flow", queueName.getSecondComponent());
-    Assert.assertEquals("flowlet", queueName.getThirdComponent());
+    Assert.assertEquals(Constants.DEFAULT_NAMESPACE, queueName.getFirstComponent());
+    Assert.assertEquals("app", queueName.getSecondComponent());
+    Assert.assertEquals("flow", queueName.getThirdComponent());
+    Assert.assertEquals("flowlet", queueName.getFourthComponent());
     Assert.assertEquals("out", queueName.getSimpleName());
 
     // convert to URI and back, verify again
     queueName = QueueName.from(queueName.toURI());
     Assert.assertTrue(queueName.isQueue());
     Assert.assertFalse(queueName.isStream());
-    Assert.assertEquals("app", queueName.getFirstComponent());
-    Assert.assertEquals("flow", queueName.getSecondComponent());
-    Assert.assertEquals("flowlet", queueName.getThirdComponent());
+    Assert.assertEquals(Constants.DEFAULT_NAMESPACE, queueName.getFirstComponent());
+    Assert.assertEquals("app", queueName.getSecondComponent());
+    Assert.assertEquals("flow", queueName.getThirdComponent());
+    Assert.assertEquals("flowlet", queueName.getFourthComponent());
     Assert.assertEquals("out", queueName.getSimpleName());
 
     // convert to bytes and back, verify again
     queueName = QueueName.from(queueName.toBytes());
     Assert.assertTrue(queueName.isQueue());
     Assert.assertFalse(queueName.isStream());
-    Assert.assertEquals("app", queueName.getFirstComponent());
-    Assert.assertEquals("flow", queueName.getSecondComponent());
-    Assert.assertEquals("flowlet", queueName.getThirdComponent());
+    Assert.assertEquals(Constants.DEFAULT_NAMESPACE, queueName.getFirstComponent());
+    Assert.assertEquals("app", queueName.getSecondComponent());
```

Comment:
We should change this `get{First|Second|Third|Fourth|Fifth|Sixth|Seventh|Eight|*}Component()` to something sensible.